ui/src/wizard/AzureWizard.ts

```diff
@@ -42,8 +41,6 @@ export class AzureWizard<T> {
     }
 
     public async execute(actionContext: IActionContext): Promise<T> {
-        ext.outputChannel.show(true);
```

Comment:
Interesting - so this is going to affect every single wizard. Do we have a plan for addressing that? I'm most concerned about Cosmos since that'll probably ship the soonest. Might be easy enough to fix the one or two wizards

Reply:
I think it's fine. The wizard isn't used all that often anyways. Just submit a PR to Cosmos to call outputChannel.show